doc/src/manual/arrays.md

```diff
@@ -1028,33 +1028,57 @@ It is sometimes useful to perform element-by-element binary operations on arrays
 sizes, such as adding a vector to each column of a matrix. An inefficient way to do this would
 be to replicate the vector to the size of the matrix:
 
-```julia-repl
-julia> a = rand(2,1); A = rand(2,3);
+```jldoctest broadcasting
+julia> a = [1, 2]; A = [10 20 30; 40 50 60]
 
-julia> repeat(a,1,3)+A
-2×3 Array{Float64,2}:
- 1.20813  1.82068  1.25387
- 1.56851  1.86401  1.67846
+julia> repeat(a, 1, 3) + A
+2×3 Matrix{Int64}:
+ 11  21  31
+ 42  52  62
 ```
 
-This is wasteful when dimensions get large, so Julia provides [`broadcast`](@ref), which expands
-singleton dimensions in array arguments to match the corresponding dimension in the other array
+This is wasteful when dimensions get large, so Julia provides
+[`broadcast`](@ref), which aligns dimensions in array arguments from left to
+right[^2], fills lacking dimensions with a singleton dimension, expands
```

Comment:
Describing these as left and right seems a little strange to me. There are arguments left and right of + but this isn't what's meant. The dimensions are numbered (in many contexts) but do we talk of the 1st one as left-most in other contexts?

Maybe it should say "aligns dimensions in array arguments starting with the first"?